stdlib/public/SwiftShims/GlobalObjects.h

```diff
@@ -24,7 +24,10 @@
 #include "Visibility.h"
 
 #ifdef __cplusplus
-namespace swift { extern "C" {
+#ifndef __swift__
```

Comment:
Can you document this flag somewhere?

Reply:
This is the standard __swift__ version number.